File: python/scripts/get_invoice_pdf.py
import os
from travelperk_http_python.builder.builder import build
from utils import write_output
travelperk = build(os.environ["SANDBOX_API_KEY"], True)
invoice_pdf = travelperk.expenses().invoices().pdf("CR-01-2")
write_output("invoice_pdf", [
    invoice_pdf,
])

File: php/scripts/get_invoice_pdf.php
<?php
require_once(__DIR__."/../vendor/autoload.php");
require_once("utils.php");
use Namelivia\TravelPerk\ServiceProvider;
$travelperk = (new ServiceProvider())->build(getenv("SANDBOX_API_KEY"), True);
$invoicePDF = $travelperk->expenses()->invoices()->pdf("CR-01-2");
write_output("invoice_pdf", [
	$invoicePDF
]);
